roles/pulibrary.pas/defaults/main.yml

```diff
@@ -4,6 +4,9 @@ pas_db_user: 'pas'
 apache_app_path: '/opt/pas'
 pas_db_name: 'pas'
 pas_db_password: '{{ pas_password }}'
+pas_server_name: '{{ application_server | default("localhost") }}'
```

Comment:
@kayiwa The variables are some application level config details that seemed best automate as  part of the build. I son et them to default to the defaults for vagrant development. Apparently the craftcms app needs to be told the server it is on and have two different URLs hardcoded to what the public URL of the site will be.